Implement new api and podcast.

dineshdb opened the pull request.

dineshdb committed:
Fix http image url to https
	modified:   src/shuvayatra-channel.html

dineshdb committed:
Upgrade New api for podcast [on progress]
	modified:   src/behaviors/global-behavior.html
	modified:   src/shuvayatra-channel.html
	modified:   src/shuvayatra-radio.html
	modified:   src/shuvayatra-slider-block.html

dineshdb committed:
Implement /radio page and update to new podcast api.
	new file:   src/radio-slider.html
	modified:   src/shuvayatra-radio.html

dineshdb committed:
Implement shuvayatra channel detail and upgrade to new api.

dineshdb committed:
Implement play queue.
	modified:   src/shuvayatra-channel.html
	modified:   src/shuvayatra-mini-player.html

dineshdb committed:
Implement mini-player skip
	modified:   src/shuvayatra-channel.html
	modified:   src/shuvayatra-mini-player.html

tyohan commented:
Looks good, but I notice when you play a podcast, the play icon is on pause state then when you click it's not change the icon's state. You need to click it twice to get the right icon's state.

dineshdb commented:
Yeah, I'm aware of the bug. Just don't have time to look into it.

dineshdb committed:
Fix Play-pause status icon
Add Remove from queue
Add Autoplay next item in queue after completion of playback
	modified:   src/radio-slider.html
	modified:   src/shared-styles.html
	modified:   src/shuvayatra-channel.html
	modified:   src/shuvayatra-mini-player.html

dineshdb commented:
@tyohan I've fixed the bug, PR is ready to be merged. Please have a look.

dineshdb committed:
Improve ui
	modified:   src/shared-styles.html
	modified:   src/shuvayatra-channel.html
	modified:   src/shuvayatra-mini-player.html

dineshdb committed:
Fix padding in queue list items
Add selection of currently playing item in queue.
	modified:   src/shared-styles.html
	modified:   src/shuvayatra-mini-player.html
